server/router/profile.js

```diff
@@ -0,0 +1,56 @@
+const express = require('express');
+
+const Profile = require('../models/profileModel')
+
+const router = new express.Router();
+
+//An endpoint to fetch all of the user profiles
+router.get('/profiles', async (req,res) => {
+    try {
+        const profiles = await Profile.find();
+        res.json(profiles)
+    } catch(err) {
+        console.log(`Exception caught --------> ${err}`)
+        res.status(500).send(err);
+    }
+})
+
+
+//To register a new user
+router.post('/signup', async (req,res) => {
+    try {
+        const profile = new Profile(req.body);
+        await profile.save();
+        res.status(201).json(profile);
+    } catch(e) {
+        console.log(`Exception caught --------> ${e}`)
+        res.status(500).send(e);
+    }
+})
+
+//To authenticate a user
+router.post('/login', async (req,res) => {
+    try {
+        console.log(req.body)
+        if(req.body.username && req.body.password) {
```

Comment:
comment from meeting about what if no username/passwsord

Reply:
@meyeralex31  Thanks fixed it now https://github.com/meyeralex31/csci550/pull/1/commits/d3776e95720c63e060e81d2879e4e94ec4b2d176